types/controller.go

```diff
@@ -55,7 +59,9 @@ type Controller struct {
 // NewController create a new connector SDK controller
 func NewController(credentials *auth.BasicAuthCredentials, config *ControllerConfig) *Controller {
 
-	invoker := NewInvoker(config.GatewayURL,
+	gatewayFunctionPath := checkAsyncRoute(config)
+
+	invoker := NewInvoker(gatewayFunctionPath, //config.GatewayURL,
 		MakeClient(config.UpstreamTimeout),
 		config.PrintResponse)
```

Comment:
What is the response that gets printed here if the function is invoked asynchronously?

Reply:
Its as we would call it in the general case I get 202 Like in the PR description, I believe this part:
```
2019/06/10 13:24:12 connector-sdk got result: [202] faas-request => figlet (0) bytes
[202] faas-request => figlet
```